### PR TITLE
Katas UI string review - Deutsch algorithm

### PR DESCRIPTION
**Pull request by SoniaLopezBravo**

### CODE REVIEW

File: katas/content/deutsch_algo/index.md [hunk 1]
```diff
@@ -120,24 +120,24 @@ $$U_f \left( \frac{1}{\sqrt2} \big( \ket{0} + \ket{1} \big) \right)
...
 
 1. Start with a qubit in the $\ket{0}$ state.
 2. Apply the $H$ gate to the qubit.
 3. Apply the oracle.
 4. Apply the $H$ gate to the qubit again.
-5. Measure the qubit: if it is in the $\ket{0}$ state, the function is constant, otherwise it is variable.
+5. Measure the qubit: if it's in the $\ket{0}$ state, the function is constant, otherwise it's variable.
 
-Note that this algorithm requires only **1** oracle call, and always produces the correct result (is deterministic).
+Note that this algorithm requires only **1** oracle call, and always produces the correct result (it's deterministic).
```
Comment (tcNickolas) [inline katas/content/deutsch_algo/index.md:133]:
```suggestion
Note that this algorithm requires only **1** oracle call, and always produces the correct result (the algorithm is deterministic).
```

**Review by tcNickolas — CHANGES_REQUESTED** [review]

**Review by tcNickolas — APPROVED** [review]

File: katas/content/deutsch_algo/implement_algo/solution.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ Follow the algorithm as outlined in the previous section:
 2. Apply the $H$ gate to the qubit.
 3. Apply the oracle. The syntax for applying the oracle is the same as for applying any other gate or operation.
 4. Apply the $H$ gate to the qubit again.
-5. Measure the qubit: if the measurement result is `Zero`, the function is constant, otherwise it is variable.
-Since we need to return `true` if the function is constant, we can just return the result of comparing the measurement result with `Zero`.
+5. Measure the qubit: if the measurement result is `Zero`, the function is constant, otherwise it's variable.
```
Comment (DmitryVasilevsky) [inline katas/content/deutsch_algo/implement_algo/solution.md:7]: I see a lot of shortening in these PRs (it is -> it's, etc.) Is this considered a better style?

**Review by DmitryVasilevsky — APPROVED** [review]

File: katas/content/deutsch_algo/implement_algo/solution.md (hunk 2 above)
Comment (tcNickolas) [reply katas/content/deutsch_algo/implement_algo/solution.md:7]: Not universally :-) Technical books and other formal writing scenarios discourage use of contractions. @SoniaLopezBravo  I assume there is a guideline somewhere about Microsoft position on contractions in learning materials?

Comment (SoniaLopezBravo) [reply katas/content/deutsch_algo/implement_algo/solution.md:7]: Yes, using contractions is part of the [Microsoft Writing Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36389). There are many other guidelines to help create a casual tone.

**Pull request merged**